Resources/views/Collector/db.html.twig

```diff
@@ -186,55 +162,83 @@
+        <p>Second Level Cache is not enabled.</p>
+    </div>
+    {% else %}
+        {% if collector.cacheCounts %}
+            {% if profiler_markup_version == 1 %}
+                {{ include('WebProfilerBundle:Profiler:table.html.twig', { data: collector.cacheCounts }, with_context = false) }}
+            {% else %}
+                <div class="metrics">
+                    <div class="metric">
+                        <span class="value">{{ collector.cacheCounts.hits }}</span>
+                        <span class="label">Hits</span>
+                    </div>
+
+                    <div class="metric">
+                        <span class="value">{{ collector.cacheCounts.misses }}</span>
+                        <span class="label">Misses</span>
+                    </div>
+
+                    <div class="metric">
+                        <span class="value">{{ collector.cacheCounts.puts }}</span>
+                        <span class="label">Puts</span>
+                    </div>
+                </div>
+            {% endif %}
 
-        {% if collector.cacheRegions.hits %}
-            <h3>Number of cache hits</h3>
-            {% include 'WebProfilerBundle:Profiler:table.html.twig' with {data: collector.cacheRegions.hits} only %}
-        {% endif %}
+            {% if collector.cacheRegions.hits %}
+                <h3>Number of cache hits</h3>
+                {{ include('WebProfilerBundle:Profiler:table.html.twig', { data: collector.cacheRegions.hits }, with_context = false) }}
+            {% endif %}
 
-        {% if collector.cacheRegions.misses %}
-            <h3>Number of cache misses</h3>
-            {% include 'WebProfilerBundle:Profiler:table.html.twig' with {data: collector.cacheRegions.misses} only %}
-        {% endif %}
+            {% if collector.cacheRegions.misses %}
+                <h3>Number of cache misses</h3>
+                {{ include('WebProfilerBundle:Profiler:table.html.twig', { data: collector.cacheRegions.misses }, with_context = false) }}
+            {% endif %}
 
-        {% if collector.cacheRegions.puts %}
-            <h3>Number of cache puts</h3>
-            {% include 'WebProfilerBundle:Profiler:table.html.twig' with {data: collector.cacheRegions.puts} only %}
+            {% if collector.cacheRegions.puts %}
+                <h3>Number of cache puts</h3>
+                {{ include('WebProfilerBundle:Profiler:table.html.twig', { data: collector.cacheRegions.puts }, with_context = false) }}
+            {% endif %}
+        {% else %}
+            <div class="empty">
+                <p>Second level cache information is not available.</p>
+            </div>
         {% endif %}
-    {% else %}
-        <p>
-            <em>No cache.</em>
-        </p>
     {% endif %}
 
-    <h2>Mapping</h2>
+    <h2>Entities Mapping</h2>
 
     {% for manager, classes in collector.entities %}
-        <h3>Manager <em>{{ manager }}</em></h3>
+        <h3>{{ manager }} <small>entity manager</small></h3>
```

Comment:
I tried ... but I don't know how to obtain the complete list of managers to check if there is only one.